### DISCUSSION TITLE
Set Bossbar Color (for various features)

### DISCUSSION THREAD

**Discussion by Cokenpizza** [Zrips/CMI#7059] [Enhancement]
CMI uses bossbars for a variety of features, like the compass, vanish, combat timer, and HP bars.  I would love to see an option in the config to change the color that each boss bar uses to better fit servers that use custom resource packs.

**Comment by mrfloris** [comment]
<img width="1211" alt="Screen-Shot 2022-09-25 at 18 07 32" src="https://user-images.githubusercontent.com/28841349/192153481-32bfd44c-b911-48f0-a37a-f4934e91d9e2.png">
I think for compass you can set the color; `  Color: '&7'` or is this the color for the text of the bossbarmsg? 

It would be nice, I see this question come up on discord once in a while. at least a few times per month.

**Reply by CloudeLecaw** [reply]
Well i think he meant the color of the backround stripes of things where the bossbar is used like
combat enable/disbale  or the vanish command and a default (for example if no color is defined in bossbarmsg)

Examples of the strip element that wished to be edded you see here..
may they show better then worlds what might be wanted here :)
![image](https://user-images.githubusercontent.com/12080017/207011917-1a47bce6-72df-4062-bef3-874bbdd733d3.png) ![image](https://user-images.githubusercontent.com/12080017/207013271-90294efa-acdf-400e-8f10-2188836b1ee4.png)


at the moment we just can design this with the bossbarmeassage command in cmi :3
